test/haskell-language-server/stack.nix

```diff
@@ -8,7 +8,11 @@ let
       rev = "e390da54a90f2c6c02ce836a27f2d4c33a71f607";
       sha256 = "0s77r79zn6q42k3biylni81vhn1yr8r1mqn8pj9kgggh16xw229b";
     };
-    projectFileName = "stack-${buildPackages.haskell-nix.compiler.${compiler-nix-name}.version}.yaml";
+    projectFileName = "stack-${
```

Comment:
Then I think you can drop this change too if you like. Less weirdness for future us.